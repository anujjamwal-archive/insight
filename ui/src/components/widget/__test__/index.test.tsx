import * as React from "react";
import { shallow } from "enzyme";
import Widget from "..";
import { Text } from "../../../vendor/elements";

describe("Widget", () => {
  it("should render", () => {
    const column = shallow(
      <Widget title="Some Title" height={10} width={10} />
    );

    expect(column).toMatchSnapshot();
  });

  it("should render with children", () => {
    const column = shallow(
      <Widget title="Some Title" height={10} width={10}>
        <Text>Hello</Text>
      </Widget>
    );

    expect(column).toMatchSnapshot();
  });
});
