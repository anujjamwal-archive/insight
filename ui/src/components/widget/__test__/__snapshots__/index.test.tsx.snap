// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widget should render 1`] = `
<Container
  height={10}
  plane="content"
  width={10}
>
  <Column
    mainAxisAlignment="flex-start"
    mainAxisSize="max"
  >
    <Row
      crossAxisAlignment="center"
      crossAxisSize="40px"
      mainAxisAlignment="flex-start"
    >
      <Text>
        Some Title
      </Text>
    </Row>
    <Row
      crossAxisSize="max"
      mainAxisAlignment="flex-start"
    />
  </Column>
</Container>
`;

exports[`Widget should render with children 1`] = `
<Container
  height={10}
  plane="content"
  width={10}
>
  <Column
    mainAxisAlignment="flex-start"
    mainAxisSize="max"
  >
    <Row
      crossAxisAlignment="center"
      crossAxisSize="40px"
      mainAxisAlignment="flex-start"
    >
      <Text>
        Some Title
      </Text>
    </Row>
    <Row
      crossAxisSize="max"
      mainAxisAlignment="flex-start"
    >
      <Text>
        Hello
      </Text>
    </Row>
  </Column>
</Container>
`;
